docs/logto-oss/deployment-and-configuration.mdx

```diff
@@ -93,6 +93,47 @@ You are all set. Open the browser and visit `https://admin.your-domain.com`, you
 
 For production, you may use Docker to containerize Logto. You can find the Dockerfile in the root directory of the project. If you want to run multiple instances of Logto, for instance, deploy Logto in a Kubernetes cluster, There are some additional steps you need to take.
 
+### Database setup \{#database-setup}
```

Comment:
Hi, we already have the DB setup covered in a previous session: https://docs.logto.io/logto-oss/get-started-with-oss#local

Reply:
I thought it would be useful to mention these things here since this section discusses the Kubernetes setup. Do you think it would instead be better to just update the docs to point to the docker section for db setup ?

Reply:
ok